slime.el

```diff
@@ -3672,14 +3672,14 @@ alist but ignores CDRs."
 ;;;; Edit definition
 
 (defun slime-push-definition-stack ()
-  "Add point to find-tag-marker-ring."
-  (require 'etags)
-  (ring-insert find-tag-marker-ring (point-marker)))
+  "Add point to find-tag-marker-stack."
+  (require 'xref)
```

Comment:
While we’re at it. Why require at runtime? Can’t we just move this dependency to the beginning of the file like every other dependency?

Reply:
Seems to be an artifact of something from the past... the runtime-require of etags is unnecessary since it's a dependency ether way

Reply:
Yeah it's from when etags was loaded only when needed on XEmacs